tsrc/test/cli/test_status.py

```diff
@@ -32,7 +32,7 @@ def test_status_dirty(tsrc_cli: CLI, git_server: GitServer,
     git_server.push_file("foo/bar", "CMakeLists.txt")
     manifest_url = git_server.manifest_url
     tsrc_cli.run("init", manifest_url)
-    workspace_path.joinpath("foo", "bar", "CMakeLists.txt").write_text("DIRTY FILE")
+    (workspace_path / "foo/bar/CMakeLists.txt").write_text("DIRTY FILE")
```

Comment:
foo/bar is converted automatically to what is required by the platform?

Reply:
Not sure what you mean.